swal-forms.js

```diff
@@ -59,7 +62,7 @@
       return inputArray
               .filter(uncheckedRadiosAndCheckboxes)
               .map(toValuableAttrs)
-              .reduce(toSingleObject)
+              .reduce(toSingleObject,{})
```

Comment:
I don't see how adding this `{}` would add value. Input for the `reduce` call should be either an empty array or an array filled with objects. This might solve an issue, but it's not clear to me which one.